mtags-interfaces/src/main/java/scala/meta/pc/OutlineFiles.java

```diff
@@ -0,0 +1,16 @@
+package scala.meta.pc;
+import java.util.List;
+
+public interface OutlineFiles {
+
+  /**
+   * Will this outline compilation be substitute for build server's compilation.
+   * Used if the first compilation using build server is unsuccessful.
+   */
+  boolean isFirstCompileSubstitute();
+
+  /**
+   * Files that should be outline compiled before calculating result. 
+   */
+  List<VirtualFileParams> files();
```

Comment:
Alternatively for the files that need to be read for the disk we can send just the URI.

Reply:
We would need to read it via buffers, which would require another interface :thinking: 

Let's leave as is and fix if it shows up as a problem